### PR TITLE
IN-194 get code

### PR DESCRIPTION
**Pull request by jmackown**
Small function to get a lpa/actor/code combo either by lpa/actor or code. Used by a number of other methods, plus a good opportunity to build up test suite and mock db.

Tried to do this yesterday was killed by AWS_DEFAULT_REGION. It was using my saved creds locally, so it worked fine here but failed on CI because it has no creds. Gaaahhhh.

### CODE REVIEW

File: lambda_functions/v1/tests/code_generator/conftest.py [hunk 1]
```diff
@@ -0,0 +1,161 @@
+import json
+
+import boto3
+import pytest
+from moto import mock_dynamodb2
+import os
+
+from lambda_functions.v1.functions.lpa_codes.app.api import code_generator
+
+# from lambda_functions.v1.tests.code_generator.test_data import test_data
```
Comment (jamesrwarren) [inline lambda_functions/v1/tests/code_generator/conftest.py:10]: Want to get rid of your commented code or is there a reason it's in here?

File: lambda_functions/v1/tests/code_generator/conftest.py [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+                    "Projection": {"ProjectionType": "ALL"},
+                    "ProvisionedThroughput": {
+                        "ReadCapacityUnits": 5,
+                        "WriteCapacityUnits": 5,
+                    },
+                }
+            },
+        ]
+
+        active_index = [
```
Comment (jamesrwarren) [inline lambda_functions/v1/tests/code_generator/conftest.py:69]: Are we doing indexes in terraform or in the code or should we be doing both? Not sure myself.

File: lambda_functions/v1/tests/code_generator/conftest.py (hunk 1 above)
Comment (jmackown) [reply lambda_functions/v1/tests/code_generator/conftest.py:10]: Forgot about it, will fix in next PR

File: lambda_functions/v1/tests/code_generator/conftest.py (hunk 2 above)
Comment (jmackown) [reply lambda_functions/v1/tests/code_generator/conftest.py:69]: Terraform. This is just for the mock version. Once I have it 'right' here I will match up Terraform.

**Pull request merged**